File: math_py/math2.py
# 30 이상의 두 자리 자연수 중 짝수의 개수
count = 0
for i in range(30, 100):
    if(i % 2 == 0):
        count += 1
        print(i)
print("짝수의 개수는",count,"개입니다.")
print()

# 20 이상 70 미만인 두 자리 수 중 5의 배수의 개수
count = 0
for i in range(20, 70):
    if(i % 5 == 0):
        count += 1
        print(i)
print("5의 배수의 개수는",count,"개입니다.")

File: math_java/math2.java
package math_java;

public class math2 {
	public static void main(String[] args) {
    // 3번 프린트 문제 7번
		int count = 0;
		for(int i = 30; i <= 99; i++) {
			if(i % 2 == 0) {
				count++;
			}
		}
		System.out.println("짝수의 개수 : " + count + "가지");
		
		// 3번 프린트 문제 8번
			int cnt = 0;
			for(int i = 20; i < 70; i++) {
				if(i % 5 == 0) {
					cnt++;
				}
			}
			System.out.println("5의 배수의 개수 : " + cnt + "가지");
     }
  }